Microsoft.Toolkit.Uwp.UI.Controls/ImageEx/ImageExBase.Placeholder.cs

```diff
@@ -52,6 +50,29 @@ public ImageSource PlaceholderSource
             set { SetValue(PlaceholderSourceProperty, value); }
         }
 
+        private static void PlaceholderSourceChanged(DependencyObject d, DependencyPropertyChangedEventArgs e)
+        {
+            var control = d as ImageExBase;
+
+            if (e.OldValue == null || e.NewValue == null || !e.OldValue.Equals(e.NewValue))
+            {
+                if (control is RoundImageEx round)
```

Comment:
From what I know, C# 7 is not currently available for Continuous Integration. Please change to use C# 6 features such as `as`.